pkg/cephfs/driver.go

```diff
@@ -28,7 +28,7 @@ import (
 const (
 
 	// version of ceph driver
-	version = "1.0.0"
+	version = "1.1.0"
```

Comment:
need to update driver version for rbd  also
Note: we may need to do cleanup to have a single variable for the version

Reply:
You could consider using a method similar to what Rook uses to set the version automatically using `go` linker flags at build time. That would prevent any version updates from being missed due to human error. 

You can look for `LDFLAGS` in Rook's `Makefile` and see how it propagates into `build/makelib/golang.mk`